yoti-sdk-api/src/main/java/com/yoti/api/client/Anchor.java

```diff
@@ -5,14 +5,40 @@
 
 public interface Anchor {
 
+    /**
+     * The type of Anchor - most likely either SOURCE or VERIFIER, but its possible that new Anchor types will be added in future
+     *
+     * @return String, naming the type of the anchor
+     */
     String getType();
 
+    /**
+     * Describes how the attribute value was obtained, for example this might be 'OCR' or 'NFC'
```

Comment:
This makes sense for sources, but the subtype for Verifiers isn't really how the attribute value was obtained is it? E.g. we can have a Verifier value of `YOTI_IDENTITY` with a subtype of `Callcredit`, but we are just verifying with Callcredit, not obtaining the value from them.